nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -18,29 +18,53 @@
-|                     | `local_context_schema`                  | `local_config_context_schema`                  |
-| VLAN                | `group`                                 | `vlan_group`                                   |
+| Model                   | Renamed Field                           | New Name                                       |
+|-------------------------|-----------------------------------------|------------------------------------------------|
+| Cluster                 | `group`                                 | `cluster_group`                                |
+|                         | `type`                                  | `cluster_type`                                 |
+| ComputedField           | `computedfield`                         | `computed_fields`                              |
+| ConfigContext           | `schema`                                | `config_context_schema`                        |
+|                         | `configcontext`                         | `config_contexts`                              |
+|                         | `configcontext`                         | `config_contexts`                              |
+| ConfigContextSchema     | `configcontextschema`                   | `config_context_schemas`                       |
+|                         | `device_set`                            | `dcim_device_related`                          |
+|                         | `virtualmachine_set`                    | `virtualization_virtualmachine_related`        |
+| CustomFieldChoice       | `field`                                 | `custom_field`                                 |
+|                         | `choices`                               | `custom_field_choices`                         |
+| CustomLink              | `customlink`                            | `custom_links`                                 |
+| Device                  | `device_role`                           | `role`                                         |
+|                         | `local_context_data`                    | `local_config_context_data`                    |
+|                         | `local_context_data_owner_content_type` | `local_config_context_data_owner_content_type` |
+|                         | `local_context_data_owner_object_id`    | `local_config_context_data_owner_object_id`    |
+|                         | `local_context_schema`                  | `local_config_context_schema`                  |
+| DynamicGroup            | `dynamicgroup`                          | `dynamic_groups`                               |
+| ExportTemplate          | `exporttemplate`                        | `export_templates`                             |
+| GitRepository           | `gitrepository`                         | `git_repositories`                             |
+| ImageAttachment         | `imageattachment`                       | `image_attachments`                            |
```

Comment:
Aren't these (and several others in this table) changes to the reverse related_name on the other model, not changes on this model itself? In other words, shouldn't this be something more like:

```suggestion
| ContentType             | `dynamicgroup`                          | `dynamic_groups`                               |
| ContentType             | `exporttemplate`                        | `export_templates`                             |
| SecretsGroup            | `gitrepository`                         | `git_repositories`                             |
| ContentType             | `imageattachment`                       | `image_attachments`                            |
```